docs/content/v2.20/architecture/transactions/concurrency-control.md

```diff
@@ -309,11 +309,7 @@ Note that the retries will not be performed in case the amount of data to be sen
 
 ## Wait-on-Conflict
 
-{{< note >}}
-
-Wait-on-conflict is [Tech Preview](/preview/releases/versioning/#feature-availability).
-
-{{</note >}}
+Wait-on-conflict feature is in {{<badge/tp>}}.
```

Comment:
Is it EA in Preview? GA in Stable?

Reply:
TP - 2.19.1 
EA - 2.20
GA -2024.1
https://yugabyte.atlassian.net/jira/polaris/projects/IDEA/ideas/view/3573079?selectedIssue=IDEA-463&issueViewSection=comments

Reply:
modified it.